db/migrate/20240730090211_remove_proxy_byos_boolean.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveProxyByosBoolean < ActiveRecord::Migration[6.1]
+  def up
+    safety_assured { remove_column :systems, :proxy_byos }
```

Comment:
So if the `proxy_byos` column is removed what happens if the sibling server sends that value?

Reply:
The code that receives the instance sharing data needs to convert `proxy_byos=true` to `proxy_byos_mode=byos`.

Reply:
we would need to adapt a regsharing method on the controller, see  [this comment](https://github.com/SUSE/rmt/pull/1194#discussion_r1699604905)

I'm fine keeping the column and dropping once all siblings are OK